packages/editor/src/components/browser/useBrowser.ts

```diff
@@ -20,6 +21,7 @@ export type BrowserType =
 export type UseBrowserImplReturnValue = Omit<Tab, 'id'> & {
   id: BrowserType;
   accept: () => string;
+  icon?: IvyIcons;
```

Comment:
if all browser now support an icon, you can also make this required -> remove optional

Reply:
Thank you, I also considered that. However, I left it optional because, as far as I could see, Anais has not designed an icon for, for example, the Category Path Chooser or the Table Column Browser yet. I will make it required, though, and temporarily use an icon that seems reasonably fitting to me for these browsers.